pkgs/development/perl-modules/strip-nondeterminism/default.nix

```
@@ -18,7 +18,8 @@ buildPerlPackage rec {
   doCheck = false;
 
   nativeBuildInputs = lib.optionals stdenv.isDarwin [ shortenPerlShebang ];
-  buildInputs = [ ArchiveZip ArchiveCpio file ];
+  buildInputs = [ ArchiveZip ArchiveCpio ];
+  propagatedBuildInputs = [file];
```

```suggestion
  propagatedBuildInputs = [ file ];
```

---

```suggestion
  propagatedNativeBuildInputs = [ file ];
```
Needs to be ran on host machine during the build.